client/scripts/main.js

```diff
@@ -371,6 +371,28 @@ var __listen_to_the_ft = (function(){
 
 	}());
 
+	function getQueryParameters(parameter){
```

Comment:
You can use this instead if preffered - `var urlParams = new URLSearchParams(window.location.search);`